lighttpd/datadog_checks/lighttpd/lighttpd.py

```diff
@@ -60,28 +59,27 @@ class Lighttpd(AgentCheck):
 
     RATES = {b'Total kBytes': 'lighttpd.net.bytes_per_s', b'Total Accesses': 'lighttpd.net.request_per_s'}
 
-    def __init__(self, name, init_config, agentConfig, instances=None):
-        AgentCheck.__init__(self, name, init_config, agentConfig, instances)
+    HTTP_CONFIG_REMAPPER = {'user': {'name': 'username'}}
+
+    def __init__(self, name, init_config, instances=None):
+        AgentCheck.__init__(self, name, init_config, instances)
         self.assumed_url = {}
 
+        if instances is not None and 'auth_type' in instances[0]:
+            if instances[0]['auth_type'] == 'digest':
```

Comment:
Should `auth_type` be an options to add to RequestsWrapper ? No necessarily in this PR.

Reply:
So far, I've only come across one integration that uses `auth_type`. Definitely can discuss this with @ofek to see if it's something we want to incorporate in the wrapper.